test/unit/spec/v2/ion/i18nTransformer_spec.js

```diff
@@ -1490,7 +1491,14 @@ describe('v2/ion/i18nTransformer', function() {
               ]
             },
             'class': 'INFO'
-          }
+          },
+          {
+            'message': 'Invalid',
```

Comment:
Shouldn't this be `Invalid Phone Number.`? in the messages object ?

Reply:
It doesn't matter what that message is. I'm just trying to test the key mapping in `i18nTransformer`.
`'api.factors.error.sms.invalid_phone': 'oie.phone.invalid',`

Reply:
It also shows how the `message` that comes from API is not respected as long as the key-mapping exists. So i'd leave it as is. Let me know if you think otherwise.